lib/tilelivecopy.js

```diff
@@ -45,9 +47,11 @@ function tilelivecopy(srcUri, s3url, options, callback) {
         options.close = true;
         if (scheme === 'list') options.listStream = src.createZXYStream();
 
-        if (options.stats) {
-          options.transform = stats;
-        }
+        var transforms = [];
+
+        if (options.stats) transforms.push(stats);
+        if (options.type === 'mbtiles') transforms.push(MigrationStream());
+        if (transforms.length > 0) options.transform = combiner(transforms);
```

Comment:
@mapsam yep! If there's stats, we want the stats stream. If it's an mbtiles we want the migration stream. So we combine them together and throw it into the transform of tilelive-copy.